api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb

```diff
@@ -25,5 +25,5 @@ angular.module('mnoEnterprise.configuration', [])
   .constant('APP_NAME', <%= MnoEnterprise.app_name.to_json %>)
   .constant('URL_CONFIG', <%= Hash(Settings.url_config).to_json %>)
   .constant('DEVISE_CONFIG', <%=
-      Hash(Settings.devise).merge(min_password_length: Devise.password_length.min).to_json
+      Hash(Settings.devise).merge(min_password_length: Devise.password_length.min, timeout_in: (Devise.timeout_in || 0).seconds).to_json
```

Comment:
```ruby
2.3.3 :006 > 5.minutes.to_i
 => 300 
2.3.3 :007 > nil.to_i
 => 0
```

Reply:
Actually you could leave: `timeout_in: Devise.timeout_in` which will be translated to the number of second or to `null` when passed to `to_json`. You can then handle the default  in the frontend (see my review)